### PR TITLE
In d payables

### PR DESCRIPTION
**Pull request by gowthamd3**
---
Please check the following conditions for your PR.

- [ ] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [ ] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

**Comment by sriyen-msft** [comment]
@gowthamd3 Please resolve the branch conflicts by syncing your local repo with remote repo to get your PR merged.

**Comment by gowthamd3** [comment]
Hi Srikanth,

I'm unable to merge can you help me in this regards.

**Comment by sriyen-msft** [comment]
> Hi Srikanth,
> 
> I'm unable to merge can you help me in this regards.

Please sync your local repo with remote repo and push the changes

**Comment by sriyen-msft** [comment]
> > Hi Srikanth,
> > I'm unable to merge can you help me in this regards.
> 
> Please sync your local repo with remote repo and push the changes

@gowthamd3 Please let me know if you still seeing any issues

**Review by sriyen-msft — APPROVED** [review]

File: certified-connectors/IN-D Payables/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1,3 +1,808 @@
+
```
Comment (sriyen-msft) [inline certified-connectors/IN-D Payables/apiDefinition.swagger.json:1]: @gowthamd3 Please remove this empty lines

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/IN-D Payables/apiDefinition.swagger.json (hunk 1 above)
Comment (gowthamd3) [reply certified-connectors/IN-D Payables/apiDefinition.swagger.json:1]: @sriyen-msft removed empty lines

Comment (sriyen-msft) [reply certified-connectors/IN-D Payables/apiDefinition.swagger.json:1]: @gowthamd3 Empty lines are still present

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/IN-D Payables/apiDefinition.swagger.json [hunk 2]
```diff
@@ -1,3 +1,805 @@
...
+      "name": "x-api-key"
+    }
+  },
+  "security": [
+    {
+      "API Key": []
+    }
+  ],
+  "tags": []
+}
```
Comment (sriyen-msft) [inline certified-connectors/IN-D Payables/apiDefinition.swagger.json:802]: @gowthamd3 Please remove below swagger code.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/IN-D Payables/apiDefinition.swagger.json (hunk 2 above)
Comment (gowthamd3) [reply certified-connectors/IN-D Payables/apiDefinition.swagger.json]: @sriyen-msft  removed the code you have mentioned

Comment (sriyen-msft) [reply certified-connectors/IN-D Payables/apiDefinition.swagger.json]: @gowthamd3 It is still present in the swagger. Please remove it

**Comment by sriyen-msft** [comment]
@gowthamd3 Please remove the duplicated swagger

File: certified-connectors/IN-D Payables/apiDefinition.swagger.json (hunk 2 above)
Comment (gowthamd3) [reply certified-connectors/IN-D Payables/apiDefinition.swagger.json]: @sriyen-msft  i couldn't find any empty lines, pls let me know at which row empty lines are present?.

File: certified-connectors/IN-D Payables/apiDefinition.swagger.json [hunk 3]
```diff
@@ -1,3 +1,789 @@
...
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.in-d.ai/privacy-policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "AI;IT Operations"
+    }
+  ],
+  "definitions": {},
 {
```
Comment (sriyen-msft) [inline certified-connectors/IN-D Payables/apiDefinition.swagger.json:787]: @gowthamd3 Swagger file is duplicated below: Please remove this..

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/IN-D Payables/apiDefinition.swagger.json (hunk 3 above)
Comment (gowthamd3) [reply certified-connectors/IN-D Payables/apiDefinition.swagger.json]: Removed duplicate file @sriyen-msft

File: certified-connectors/IN-D Payables/apiProperties.json [hunk 4]
```diff
@@ -1,4 +1,4 @@
-{
+  {
```
Comment (sriyen-msft) [inline certified-connectors/IN-D Payables/apiProperties.json:1]: @gowthamd3 The `iconBrandColor` seems different in the original file you have submitted to us in the cert. portal. That is "#048c88".

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@gowthamd3 I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

File: certified-connectors/IN-D Payables/apiProperties.json (hunk 4 above)
Comment (gowthamd3) [reply certified-connectors/IN-D Payables/apiProperties.json:1]: @sriyen-msft  done the change you mentioned.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**